mypyc/ir/module_ir.py

```diff
@@ -28,7 +28,7 @@ def __init__(
         self.functions = functions
         self.classes = classes
         self.final_names = final_names
-        # Names of C statics used for Python 3.12 type variable objects.
+        # Names of C statistics used for Python 3.12 type variable objects.
```

Comment:
I think this refers to "C statics" as in "C static variables". That terminology is used elsewhere in mypyc ([example](https://github.com/python/mypy/blob/5c38427a53f409bae068c314e119fecc77730d76/mypyc/ir/ops.py#L796-L804)), so I think this is written as intended

Reply:
Thanks for pointing that out! I'll leave it as is since "statics" seems to be the correct terminology.